js/modifiedEvents.js

```diff
@@ -0,0 +1,27 @@
+export function eventIsModified(e) {
+  return e.which > 1 ||
+    e.shiftKey ||
+    e.altKey ||
+    e.metaKey ||
+    e.ctrlKey;
+}
+
+/**
+ * Allow shift-click, ctrl-click, etc. in browsers to work as intended when they
+ * might otherwise be swallowed by event handlers
+ */
+export function handleModifiedClicks() {
+  document.addEventListener('click', e => {
+    // Ignore events that aren't modified link clicks
+    if (!e.target.closest('A') || !eventIsModified(e)) return;
+
+    // Stop processing this event
+    e.preventDefault();
+    e.stopPropagation();
+
+    // Perform the click on a new link (sans-event handlers)
+    const el = document.createElement('a');
+    el.href = e.target.href;
+    el.dispatchEvent(new MouseEvent('click', e));
```

Comment:
I would ensure that this actually works as expected on modified clicks since this fake mouseevent would potentially be asking the browser to open a tab or download a file (alt-click). Feels like it may well be blocked in some browsers

Reply:
Well... it works in Chrome.   In testing FF just now, however... not so much.  Crap. :-p  Good catch.

Reply:
Using your filter-altered-clicks as suggested. :-D